include/kalmar_math.h

```diff
@@ -53,7 +53,7 @@ extern "C" double __hc_copysign_double(double x, double y) restrict(amp);
 extern "C" __fp16 __hc_cos_half(__fp16 x) restrict(amp);
 extern "C" __fp16 __hc_cos_native_half(__fp16 x) restrict(amp);
```

Comment:
This also needs to change to __hc_native_cos_half
https://github.com/RadeonOpenCompute/ROCm-Device-Libs/blob/master/hc/src/hc_math.cl#L115